Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionConfigurationChecker.cs

```diff
@@ -94,19 +105,15 @@ private static bool ReconcileLeapMotionDefine()
         /// <param name="isLeapInProject">Bool that determines if the Leap Motion Core assets are in the project</param>
         private static void ConfigureLeapMotion(bool isLeapInProject)
         {
-            FileInfo[] leapDataProviderAsmDefFile = FileUtilities.FindFilesInAssets("Microsoft.MixedReality.Toolkit.Providers.LeapMotion.asmdef");
+            FileInfo[] leapDataProviderAsmDefFile = FileUtilities.FindFilesInAssets("MRTK.LeapMotion.asmdef");
```

Comment:
Since this isn't editing the asmdef anymore, do we care if it exists? Or could this and the `.Length` check be removed?

Reply:
The length check was added for an issue we ran into a while back with the Nuget packages, so we might need to keep it there

```
            // When MRTK is used through NuGet compiled assemblies, there will not be an asmdef file in the assets directory to configure.
            if (leapDataProviderAsmDefFile.Length == 0)
            {
                return;
            }
```